### ISSUE TITLE
`visit` fails reruns when the document contains an SVG with the `<use>` element

### ISSUE THREAD

**Issue by branson-simplethread** [cypress-io/cypress#29718]
### Current behavior

In open/GUI mode, rerunning examples that `visit` a page with an SVG `<use>` element fails with an error that the `load` event never fired. The first run succeeds, but all reruns fail unless Cypress is restarted. This occurs when using Chrome and Edge, but not Firefox or Electron.

### Test code to reproduce

https://github.com/branson-simplethread/cypress-test-tiny-svg-use-load-event

### Cypress Version

13.12.0

### Node version

20.14.0

### Operating System

macOS 14.5

### Debug Logs

```shell
https://raw.githubusercontent.com/branson-simplethread/cypress-test-tiny-svg-use-load-event/master/debug-log
```

**Comment by branson-simplethread** [comment]
As an additional piece of information for debugging, and to get around this issue while it persists: you can append a random query string to the `<use>` `href` URL (`icons.svg?<random>#icon-name`) in your test environment to prevent caching and force requests which then does allow the `load` event to fire.

**Comment by andrew-demb** [comment]
We found the related issue in chromium https://issues.chromium.org/issues/347724924

**Comment by stokrattt** [comment]
fixed in 126.0.6478.126 version chrome

**Comment by branson-simplethread** [comment]
Confirmed this has fixed [my test case](https://github.com/branson-simplethread/cypress-test-tiny-svg-use-load-event) in Chrome. Presumably this will also be resolved in Edge when it pulls in the upstream Chromium fix, so I'm going to close this issue.
